packages/seacas/libraries/ioss/src/Ioss_DatabaseIO.C

```diff
@@ -339,32 +339,50 @@ namespace Ioss {
           fmt::print(Ioss::DEBUG(), "DW: dw_wait_file_stage({});\n", bb_file.filename());
         }
 #endif
-        int dwret = dw_wait_file_stage(bb_file.filename().c_str());
-        if (dwret < 0) {
-          std::ostringstream errmsg;
-          fmt::print(errmsg, "ERROR: failed waiting for file stage `{}`: {}\n", bb_file.filename(),
-                     std::strerror(-dwret));
-          IOSS_ERROR(errmsg);
-        }
+        // Shyamali : To sort the ranks from communicator
+        std::vector<int> procs(util().parallel_size());
```

Comment:
Note that this just creates a vector of size `parallel_size` containing all zeros.